aboutme.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/reset.css">
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <h1>Kevin LaCarrubba</h1>
+
+        <nav>
+            <ul>
+                <li><a href="aboutme.html">About Me</a></li>
```

Comment:
`href="aboutMe.html`

Reply:
If you are on the `About Me` page, don't make this link clickable. Switch it from an `anchor` tag to a `span` and style it the same way. This is when using reusable classes really helps